Formula/cnosdb.rb

```diff
@@ -0,0 +1,25 @@
+class Cnosdb < Formula
+  desc "Open source distributed time series database"
+  homepage "https://www.cnosdb.com"
+  url "https://github.com/cnosdb/cnosdb/archive/refs/tags/v1.0.2.tar.gz"
+  sha256 "ae412d0944b64c9b39dc1edc66f7b6f712b85bc5afad354c12b135ae71017100"
+  license "MIT"
+  head "https://github.com/cnosdb/cnosdb.git"
+
+  livecheck do
```

Comment:
Doesn't the default livecheck work?

Reply:
It doesn't work. I have delete them. Thank you.